src/main/plugin/iso19139.ca.HNAP/present/metadata-fop.xsl

```diff
@@ -41,6 +41,26 @@
       </xsl:call-template>
     </xsl:variable>
 
+    <xsl:variable name="webMapServicesProtocols" select="/root/gui/webServiceTypes" />
+
+    <xsl:variable name="isoLanguages" select="/root/gui/isolanguages" />
+
+    <xsl:variable name="port">
+      <xsl:choose>
+        <xsl:when test="/root/gui/env/server/protocol = 'https'"><xsl:value-of select="/root/gui/env/server/securePort" /></xsl:when>
```

Comment:
Looks like 3.12 GN removed this.  
https://github.com/geonetwork/core-geonetwork/pull/5569

It this condition something that should only exists in versions prior to 3.12?

Also seems like this logic to get the site url would be cleaner if it used the "getSiteUrl" from the java classes. Seems odd to add all this logic in xslt when the existing logic seems to exists in java.

Reply:
@josegar74 can you answer this question please?

(@ianwallen I am not sure your question blocks this PR or not)

Reply:
Update the code.